bin/waagent2.0

```diff
@@ -3457,9 +3457,21 @@ class RdmaHandler(object):
         """
         Write config string to /dev/hvnd_rdma
         """
-        Log("Set /dev/hvnd_rdma")
-        self.wait_rdma_dev()
-        self.write_rdma_dev_conf()
+        skip_rdma_device = False
```

Comment:
Sorry I was not clear before - 2.2.1 (`/bin/waagent`) is the current version in master, originally based on the 2.1.x branch where it diverged from 2.0.x (`/bin/waagent2.0`). So you are working with the right version now.